active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,225 @@
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+#### Responding to clicks
+
+This example shows an app with a Breadcrumb across the top, and a content area on the rest of the Window.
+The Breadcrumb shows a list of items. Clicking on an item navigates the content back, and updates
+the Breadcrumb.
+
+```xaml
+<Grid RowDefinitions="Auto,*">
+    <Breadcrumb ItemClicked="CrumbClicked" ItemsSource="{x:Bind Items, Mode=OneWay}" />
+    
+    <ContentControl Grid.Row="1" Content="{x:Bind CurrentItem, Mode=OneWay}">
+        ...
+    </ContentControl>
+</Grid>
+```
+
+```csharp
+public sealed partial class MainPage : Page
+{
+    public ObservableCollection<Items> Items { get; }
+
+    public ItemClickEventArgs Current Item { get; set;}
+
+    void CrumbClicked(Breadcrumb sender, BreadcrumbItemClickedEventArgs e)
+    {
+        for(int i = Items.Count-1; i > e.Index; i++)
+        {
+            Items.RemoveAt(i);
+        }
+
+        CurrentItem = Items[i - 1];
+    }
+
+    void NavigateToItem(Item item)
+    {
+        Items.Add(item);
+        CurrentItem = item;
+    }
+}
+```
+
+#### Breadcrumb.ItemTemplate property
+
+Gets or sets the template to display an item
+
+The Breadcrumb ItemTemplate is used to override the default display of the
+individual items in the Breadcrumb control.
+
+The following uses the ItemTemplate so that the Breadcrumb items show the Name property of the items:
+
+```xaml
+<Breadcrumb ItemsSource='{x:Bind Items}'>
+    <Breadcrumb.ItemTemplate>
+        <DataTemplate x:DataType='BreadcrumbItem`>
+            <TextBlock Text='{x:Bind Name}/>
+        </DataTemplate>
+    </Breadcrumb.ItemTemplate>
+</Breadcrumb>
+```
+
+Notes:
+* You can use a BreadcrumbItem as the root of the DataTemplate
+* If no ItemTemplate is supplied, the Breadcrumb control will convert to a string.
+
+## Other Breadcrumb members
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | Gets or sets the content of the Breadcrumb |
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  |
+
+_Spec note: the V2 of Breadcrumb will add flyouts from chevrons to view children of a node.
+Additional properties will be added to the API to enable this functionality._
+
+# API Details
+
+**Windows.UI.Xaml.Controls namespace**
+
+```cs
+[webhosthidden]
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Int32 Index { get; };
+    Object Item { get; };
+}
+
+[webhosthidden]
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+[webhosthidden]
+unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
+{
+    Breadcrumb();
+
+    Object ItemsSource{ get; set; };
+    Object ItemTemplate{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<Breadcrumb, BreadcrumbItemClickedEventArgs> ItemClicked;
+
+    static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
```

Comment:
Should the list of visible crumbs be obtainable, with an event to handle changes in the visible set?

Reply:
Could you provide a case where this might be useful to have ? We could add it in the future if there is a need for that.

Reply:
Recommend: consider as a new feature.